api/v1alpha1/flowcollector_types.go

```diff
@@ -164,6 +164,12 @@ type FlowCollectorFLP struct {
 	// HealthPort is a collector HTTP port in the Pod that exposes the health check API
 	HealthPort int32 `json:"healthPort,omitempty"`
 
+	//+kubebuilder:validation:Minimum=1
+	//+kubebuilder:validation:Maximum=65535
+	//+kubebuilder:default:=9102
+	// PromPort is the prometheus HTTP port: this port exposes prometheus metrics
+	PromPort int32 `json:"promPort,omitempty"`
```

Comment:
Also could we rename to `PrometheusPort` ? As this is the end-user exposed config, we tend to make it more expressive

Reply:
Side question: it's the same port used for pipeline metrics and internal metrics, right? We don't deploy 2 prometheus instances?

Reply:
fixed
Yes, one Prometheus end-point for all metrics